mqtt-transport/src/main/java/com/esri/geoevent/transport/mqtt/MqttInboundTransport.java

```diff
@@ -179,110 +161,43 @@ private void receive(byte[] bytes)
 				bb.flip();
 				byteListener.receive(bb, "");
 				bb.clear();
-			}
-			catch (BufferOverflowException boe)
+				if (log.isTraceEnabled())
+				{
+					log.trace("Received bytes sent to adaptor");
+				}
+			} catch (BufferOverflowException boe)
```

Comment:
formatting - the `catch` statement needs to start a new line:
```suggestion
		} catch (Exception e)
```
```suggestion
		} catch (Exception e)
```
````
try
{
  ...
}
  catch (...)
{
  ...
}
```